amundsen_application/static/js/components/common/SearchBar/InlineSearchResults/ResultItemList/ResultItem/index.tsx

```diff
@@ -7,7 +7,7 @@ export interface ResultItemProps {
   iconClass: string;
   onItemSelect: (event: MouseEvent) => void;
   subtitle: string;
-  title: string;
+  title: React.ReactNode;
```

Comment:
This is an interesting choice. Maybe we should rename to `titleNode` to denote that we expect a node and not a string.

Reply:
Thanks will add in the next PR